horreum-web/package.json

```diff
@@ -46,7 +46,7 @@
     "uuid": "9.0.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=0 react-scripts start",
```

Comment:
```suggestion
    "start": "PORT=0 react-scripts start",
```

Why? PORT=0

Reply:
PORT=0 should bind the server to the first ephemeral port, this helps when you have more than one dev instance running

Reply:
Nice 👍